rpc/chain/client.go

```diff
@@ -118,6 +122,28 @@ func isVMError(err error) bool {
 	return false
 }
 
+func (c *ClientWithFallback) SetIsConnected(value bool) {
+	c.IsConnectedLock.Lock()
+	defer c.IsConnectedLock.Unlock()
+	c.LastCheckedAt = time.Now().Unix()
+	if !value {
+		if c.IsConnected {
+			if c.WalletNotifier != nil {
+				c.WalletNotifier(c.ChainID, "down")
+			}
+			c.IsConnected = false
+		}
+
+	} else {
+		if !c.IsConnected {
+			c.IsConnected = true
+			if c.WalletNotifier != nil {
+				c.WalletNotifier(c.ChainID, "up")
```

Comment:
These "up" and "down" strings are defined in several different places. Could we have constants defined in a single spot?